lib/datastore.js

```diff
@@ -188,15 +188,18 @@ MongoDataStore.prototype.ObjectID =  mongodb.ObjectID;
 MongoDataStore.prototype.initialize = function(callback){
 
   var _this = this;
-  mongoclient.connect (_this.config.url, _this.config.opts, function (err, database) {
+
+  mongoclient.connect (_this.config.url, _this.config.opts, function (err, client) {
 
     if (err) return callback(err);
 
-    var collection = database.collection(_this.config.collection);
+    var db = client.db(_this.config.database);
```

Comment:
Don't see this being used.

Reply:
it is used in the following line: var collection = db.collection(_this.config.collection);

Reply:
collection and client are then used going forward

Reply:
Sorry, missed that.